<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewThread</class>
 <widget class="QWidget" name="ViewThread">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="70,30">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="60">
     <item>
      <widget class="QScrollArea" name="Messages">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1004</width>
          <height>407</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QScrollArea" name="ThreadDetails">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>984</width>
              <height>387</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5"/>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="80,20">
     <item>
      <widget class="MRichTextEdit" name="Message">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="FileAttachmentsLabel">
           <property name="text">
            <string>File Attachments (Optional)</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="AddFileButton">
             <property name="text">
              <string>Add File</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="RemoveFileButton">
             <property name="text">
              <string>Remove File</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="AttachmentsList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="SendMessageButton">
         <property name="text">
          <string>Send Message</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Close Thread</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MRichTextEdit</class>
   <extends>QTextEdit</extends>
   <header>mrichtextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
